GettingStarted.md

```diff
@@ -1,80 +1,165 @@
- 3. Under `Deploy` be sure to fill out the device Username and Password fields.
- 4. Under `Quick Options` press: `Build SLN, Build APPX, then Install`.
+# 5. Preparing a Scene for Mixed Reality Content
+
+Select the "Apply Mixed Reality Scene Settings" option in the Unity Editor: 
+
+` Mixed Reality Toolkit -> Configure -> Apply Mixed Reality Scene Settings`
+
+![MRTK Editor Scene Settings](/External/ReadMeImages/MixedRealityEditorSceneSettings.png)
+
+Select all the required options for your Scene type:
+
+![MRTK Scene options](/External/ReadMeImages/MixedRealitySceneOptions.png)
+
+Alternatively, you can setup your scene manually as follows:
+
+1. Create a new Scene: `File -> New Scene`
+
+2. Remove the default `Main Camera` and `Directional Light` objects in the scene.
+
+3. Add the `MixedRealityCameraParent.prefab` (found under HoloToolkit/Input/Prefabs). Check the configured options for the Parent and child camera meet your requirements.
+
+4. Add the `Managers.prefab` (found under HoloToolkit/Input/Prefabs/Cursor), which will add a new Managers folder containing the **InputManager** Prefab and the all-important UI **EventSystem** object.
+
+5. Add the `DefaultCursor.prefab` (found under HoloToolkit/Input/Prefabs/Cursor) and add that Object to the **InputManager** Cursor parameter (to avoid it being searched for on scene start) 
+
+> **For Hololens**
+> Optionally, if you wish to enable spatial mapping in your scene for HoloLens, you can add the `SpatialMapping.prefab` (found under HoloToolkit/SpatialMapping/Prefabs) to your 'Managers' object.  Be aware that you must also enable `Spatial Perception` Capabilities: `Edit/Project Settings/Player -> Inspector -> Publishing Settings/Capabilities`.
+
+# 5 Building your project 
+The MRTK provides you a quick and easy way to generate your Unity project from a custom window, which can be found under:
+
+`Mixed Reality Toolkit -> Build Window`
+
+![MRTK Build Window](/External/ReadMeImages/MixedRealityEditorBuildWindow.png)
+
+This window offers many quick options to be able to:
+
+* Build your UWP C# solution
+* Build your project APPX
+* (HoloLens) Deploy your project to a remote device
 
 ![Build Window](/External/ReadMeImages/BuildWindow.PNG)
 
-Optionally, you can use the [default build steps if needed](https://docs.unity3d.com/Manual/windowsholographic-startup.html).
+
+
+# 5.1 Running your project for Immersive Headsets
+Unity supports running your Immersive solution direct from the editor **BUT** Only when the Mixed Reality Portal is running.
+> Unity may resolve this in the future and auto-start the portal, but it is better to have it running beforehand!
+
+1. Start the MR Portal
+2. Plug-in and wake up the headset or [start the Simulator](https://developer.microsoft.com/en-us/windows/mixed-reality/using_the_windows_mixed_reality_simulator)
+3. Make sure the "Cliffhouse" is displayed in the Headset
+4. Select the scene you want to run
+5. Hit Play in the Unity Editor
```

Comment:
we should also include `Build and Run` option in editor.

Reply:
Yikes, never use that with UWP!! Either run , or build, open project and run.
Unity has ALWAYS been awful with Build and Run with UWP

Reply:
Maybe I should add a task to streamline options in the build window for WMR or HoloLens targets